tests/testthat/test_wrapper.R

```diff
@@ -1,10 +1,29 @@
-context('Test basic hector functionality')
+context('Test basic R wrapper functionality')
 
 inputdir <- system.file('input', package = 'hector')
-sampledir <- system.file('output', package = 'hector')
 testvars <- c(ATMOSPHERIC_CO2(), RF_TOTAL(), GLOBAL_TEMP())
 dates <- 2000:2300
 
+
+test_that('Rerunning spinup produces minimal change', {
+
+    # First Hector core run
+    hc <- newcore(file.path(inputdir, "hector_rcp45.ini"), name = "RCP45", suppresslogging = TRUE)
+    run(hc, 2100)
+    dd1 <- fetchvars(hc, dates, testvars)
+
+    # Reset to a time before the start date and re run.
+    reset(hc, 0.0)
+    run(hc, 2100)
+    dd2 <- fetchvars(hc, dates, testvars)
+
+    diff <- abs((dd2$value - dd1$value) / (dd1$value + 1.0e-6))
```

Comment:
Note what `1.0e-6` is in a comment

Reply:
So this test was written by a former colleague my understanding/best guess that 1.0e-6 was a small number selected to make sure that re-running the spin-up, only produces a minimal change. As to why 1e-6 was selected instead of 1e-8 I am not sure, @crvernon is that your understanding of the test as well?

Reply:
@kdorheim yes, I would just place a comment in stating what you wrote above.